DOM_creation/Lesson2/main.js

```diff
@@ -0,0 +1,15 @@
+const ul= document.getElementById("js-ul");
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.setAttribute("href", "1.html");
+const text = document.createTextNode("これです");
+
+const img = document.createElement('img');
+img.setAttribute("src", "bookmark.png");
+img.setAttribute("alt", "ブックマーク");
```

Comment:
Comment
Since src and alt are also global attributes of img, they can be written as follows.
I like this writing style because it is highly readable.
````JavaScript
img.src = "bookmark.png";
img.alt = "ブックマーク";
````

Reply:
Thank you for your feed back,
I have refactored the codes 
Kindly check my latest commit [8c5a8c2](https://github.com/Saki-Ota/JS_practice/pull/2/commits/8c5a8c26c94e70df408e70620140caee8151c8ab)